README.md

```diff
@@ -37,4 +46,4 @@ In all cases, "sub-domain" can be `"all"`. User that have access to this sub-dom
 * `node user.js add <username> <sub-domain>`
 * `node user.js del <username> <sub-domain>`
 * `node user.js update <username> <sub-domain>`
-* `node user.js list <sub-domain>`
\ No newline at end of file
+* `node user.js list <sub-domain>`
```

Comment:
add at the end:

## Why this name?

Squanchy is Rick Sanchez' crazier friend.
[![squanchy](http://www.toonzone.net/fansites/blogicon/56871/master/1466803579.jpg)](https://www.youtube.com/watch?v=WEsqSJLeeDc)